### PR TITLE
NMS-14447: Update Jira Ticketing Plugin doc

### PR DESCRIPTION
**Pull request by emarsh1995**
- Updated and reformatted reference/ticketing/ticketer/jira.adoc
- Added requested lines to document

### All Contributors

* [x] Have you read our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [x] Have you (electronically) signed the [OpenNMS Contributor Agreement](https://cla-assistant.io/OpenNMS/opennms)?

### Contribution Checklist

* Please [make an issue in the OpenNMS issue tracker](https://issues.opennms.org) if there isn't one already.<br />Once there is an issue, please:
  1. update the title of this PR to be in the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
  4. once you've created this PR, please link to it in a comment in the JIRA issue
  Don't worry if this sounds like a lot, we can help you get things set up properly.
* **If this code is likely to affect the UI, did you name your branch with `-smoke` in it to trigger smoke tests?**
* If this is a new or updated feature, is there documentation for the new behavior?
* If this is new code, are there unit and/or integration tests?
* If this PR targets a `foundation-*` branch, does it try to avoid changing files in `$OPENNMS_HOME/etc/`?

### What's Next?

A PR should be assigned at least 2 reviewers.  If you know that someone would be a good person to review your code, feel free to add them.

If you need help making additions or changes to the documentation related to your changes, please let us know.

In any case, if anything is unclear or you want help getting your PR ready for merge, please don't hesitate to say something in the comments here, or in [the #opennms-development chat channel](https://chat.opennms.com/opennms/channels/opennms-development).

Once reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the OGP) you are welcome to merge the PR when you're ready. Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* JIRA (Issue Tracker): https://issues.opennms.org/browse/NMS-14447

### CODE REVIEW

File: docs/modules/reference/pages/ticketing/ticketer/jira.adoc [hunk 1]
```diff
@@ -2,119 +2,126 @@
...
+To install the `opennms-plugin-ticketer-jira` package, use one of the following commands:
 
 * For RPM-based distributions:
 ** `yum install opennms-plugin-ticketer-jira`
 * For Debian-based distributions:
 ** `apt-get install opennms-plugin-ticketer-jira`
 ****
 
-. Once installed, set the following property in `$OPENNMS_HOME/etc/opennms.properties.d/ticketer.properties` to enable the plugin (create the file if it does not exist):
+. Once the package has been installed, set the ticketer plugin property in `$OPENNMS_HOME/etc/opennms.properties.d/ticketer.properties` to enable the plugin:
```
Comment (mmahacek) [inline docs/modules/reference/pages/ticketing/ticketer/jira.adoc:21]:
```suggestion
. Once the package has been installed, set the ticketer plugin property in `$\{OPENNMS_HOME}/etc/opennms.properties.d/ticketer.properties` to enable the plugin:
```

File: docs/modules/reference/pages/ticketing/ticketer/jira.adoc [hunk 2]
```diff
@@ -2,119 +2,126 @@
...
 
  opennms.ticketer.plugin=org.opennms.netmgt.ticketd.OSGiBasedTicketerPlugin
```
Comment (mmahacek) [inline docs/modules/reference/pages/ticketing/ticketer/jira.adoc:23]:
```suggestion
[source, properties]
----
opennms.ticketer.plugin=org.opennms.netmgt.ticketd.OSGiBasedTicketerPlugin
 ----
```

File: docs/modules/reference/pages/ticketing/ticketer/jira.adoc [hunk 3]
```diff
@@ -2,119 +2,126 @@
...
++
+If the ticketer properties file does not exist, create it manually.
 
- feature:install jira-troubleticketer
-
-The plugin should be ready to use.
-Log in to the Karaf shell and run `opennms:jira-verify` to check your configuration.
-
-.jira.properties Options
+. Configure the plugin by enabling and disabling options in `$OPENNMS_HOME/etc/jira.properties`.
```
Comment (mmahacek) [inline docs/modules/reference/pages/ticketing/ticketer/jira.adoc:28]:
```suggestion
. Configure the plugin by enabling and disabling options in `$\{OPENNMS_HOME}/etc/jira.properties`.
```

File: docs/modules/reference/pages/ticketing/ticketer/jira.adoc [hunk 4]
```diff
@@ -2,119 +2,126 @@
...
 == Custom fields
 
-Besides the common fields creator, create date, description, or subject, you also may also need to set ticket-system proprietary fields.
-The Jira ticketing plugin provides the opportunity to define those in the OpenNMS ticket attributes, which can be overwritten with Drools rules.
+Besides the common fields (creator, create date, description, and subject), you also may also need to set custom fields.
+The Jira ticketing plugin provides the opportunity to define those in the {page-component-title} ticket attributes, which can be overwritten with https://www.drools.org/[Drools] rules.
 
-To enable the Drools ticketing integration, set the following property in `$OPENNMS_HOME/etc/opennms.properties.d/ticketer.properties`:
-(Create the file if it does not exist.)
+To enable the Drools ticketing integration, set the ticketer service layer property in `$OPENNMS_HOME/etc/opennms.properties.d/ticketer.properties`:
```
Comment (mmahacek) [inline docs/modules/reference/pages/ticketing/ticketer/jira.adoc:103]:
```suggestion
To enable the Drools ticketing integration, set the ticketer service layer property in `$\{OPENNMS_HOME}/etc/opennms.properties.d/ticketer.properties`:
```

File: docs/modules/reference/pages/ticketing/ticketer/jira.adoc [hunk 5]
```diff
@@ -2,119 +2,126 @@
...
 
 [source, properties]
 ----
 opennms.ticketer.servicelayer=org.opennms.netmgt.ticketd.DroolsTicketerServiceLayer
 ----
 
-In addition, the property in `$OPENNMS_HOME/etc/drools-ticketer.properties` must point to a `drools-ticketer-rules.drl` file:
+If the ticketer properties file does not exist, create it manually.
+
+In addition, the Drools ticketer rules property in `$OPENNMS_HOME/etc/drools-ticketer.properties` must point to a `drools-ticketer-rules.drl` file:
```
Comment (mmahacek) [inline docs/modules/reference/pages/ticketing/ticketer/jira.adoc:112]:
```suggestion
In addition, the Drools ticketer rules property in `$\{OPENNMS_HOME}/etc/drools-ticketer.properties` must point to a `drools-ticketer-rules.drl` file:
```

File: docs/modules/reference/pages/ticketing/ticketer/jira.adoc [hunk 6]
```diff
@@ -2,119 +2,126 @@
...
 
 [source, properties]
 ----
 drools-ticketer.rules-file=$OPENNMS_HOME/etc/drools-ticketer-rules.drl
 ----
 
-Finally, you must place a Drools rule file named `drools-ticketer-rules.drl` in `$OPENNMS_HOME/etc`.
+Finally, you must place a Drools rule file named `drools-ticketer-rules.drl` in the `$OPENNMS_HOME/etc` directory.
```
Comment (mmahacek) [inline docs/modules/reference/pages/ticketing/ticketer/jira.adoc:119]:
```suggestion
Finally, you must place a Drools rule file named `drools-ticketer-rules.drl` in the `$\{OPENNMS_HOME}/etc` directory.
```

**Review by mmahacek — COMMENTED** [review]
I haven't used the plugin, so I can't comment on the accuracy of the text.

**Comment by emarsh1995** [comment]
Closing PR because of an issue with the branch. Will be remaking the PR using the NMS-14447b branch.

**Pull request closed without merging**